ludwig/automl/base_config.py

```diff
@@ -132,7 +132,7 @@ def _create_default_config(
     base_automl_config["hyperopt"]["executor"]["time_budget_s"] = time_limit_s
     if time_limit_s is not None:
         base_automl_config["hyperopt"]["sampler"]["scheduler"]["max_t"] = time_limit_s
-    base_automl_config["hyperopt"]["sampler"]["search_alg"]["random_state_seed"] = random_seed
+    base_automl_config["hyperopt"]["search_alg"]["random_state_seed"] = random_seed
```

Comment:
maybe use constants for HYPEROPT and SEARCH_ALG

Reply:
done.